com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/message/Messages.java

```diff
@@ -147,7 +147,9 @@
     public static final String GATHER_STATISTICS = "Gather statistics: {0}";
     public static final String HEALTH_CHECK_CONFIGURATION = "Health check configuration: {0}";
     public static final String MAIL_API_URL = "Mail API URL: {0}";
-    public static final String CONTROLLER_OPERATIONS_TIMEOUT = "CFG_CONTROLLER_OPERATIONS_TIMEOUT: {0}";
+    public static final String CONTROLLER_OPERATIONS_TIMEOUT = "Controller operations timeout: {0}";
+    public static final String APPLICATION_ID = "Application ID: {0}";

```

Comment:
ID ? or GUID?

Reply:
GUID was consistent with getSpaceGuid(), but I think I'll rename both methods to getSpaceId() and getApplicationId() respectively, so that they're consistent with their log messages.